GoogleDataTransport/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.2.0
+- Added an API for arbitrary data persistence on storage instances.
```

Comment:
Do you think we know the eventual PR number for the change?

Reply:
Unfortunately, no.